File: C#/WebshopC#Con5/Infrastructure/DTO/ProductDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Infrastructure.DTO
{
    public class ProductDTO
    {
        public string Name { get; private set; }
        public string Description { get; private set; }
        public double Price { get; private set; }
        public int Amount { get; private set; }



        public ProductDTO(string name, string description, double price, int amount)
        {

            Name = name;
            Description = description;
            Price = price;
            Amount = amount;

        }




    }
}

File: Webshop2/Webshop2/src/DTO/ProductDTO.java
package DTO;

public class ProductDTO {

	private String name;
	private double price;

	public ProductDTO() {

	}

	public ProductDTO(String name, double price) {

		this.name = name;
		this.price = price;

	}

	public String getName() {
		return this.name;
	}

	public void setName(String name) {

		this.name = name;
	}

	public double getPrice() {
		return price;
	}

	public void setPrice(double price) {

		this.price = price;

	}

}
